packages/transmute-cli/components/ansible/roles/ipfs/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
   register: output
 
 - name: install ipfs
-  shell: "helm install $(npm root -g)/transmute-cli/components/ipfs/ --name decentralized-storage --wait"
+  shell: "helm install stable/ipfs --name decentralized-storage"
```

Comment:
Is this necessary to fix macOS?

Reply:
It causes no difference on Mac using later versions; however, Mac bugs with minikube addons on 1.9.4, so it's not targeting likely bug with helm release install timeout that could happen otherwise.  Accordingly, the removal of the `--wait` argument is what is addressing the intermittent helm release install timeout on Linux seen on the TravisCI builds.

Changing from the local Chart.yaml to the stable/ipfs is to match the logic for the elasticsearch microservice playbook, as well as capitalize on updates on the stable release channel for ipfs.